pymc3/smc/sample_smc.py

```diff
@@ -170,5 +172,6 @@ def sample_smc(
         smc.pool.join()
 
     trace = smc.posterior_to_trace()
-
+    trace.report._n_draws = smc.draws
+    trace.report._t_sampling = time.time() - t1
```

Comment:
`trace.report._n_tune = n_tune` also needs to be set

Reply:
done

Reply:
there is actually no n_tune in SMC, tuning for stage t is done based on the results from stage t-1.